src/components/menu/view-controls.jsx

```diff
@@ -15,8 +18,25 @@ import AppInfoPane from '../panes/app-info-pane'
 import tasks from '../../actions/task-definitions'
 
 
-export default class ViewControls extends React.Component {
+export class ViewControlsUnconnected extends React.Component {
+  static propTypes = {
+    isAuthenticated: PropTypes.bool.isRequired,
+  }
+
   render() {
+    const loginItem = this.props.isAuthenticated ?
+      (
+        <NotebookTaskButton task={tasks.logoutGithub}>
+          <img src={this.props.avatar} alt="" className="user-avatar" />
+        </NotebookTaskButton>
+      )
+      :
+      (
+        <NotebookTaskButton task={tasks.loginGithub}>
+          <AccountCircle />
```

Comment:
I would just use the text/link "Login" or "Sign in" here instead of an account circle, which is pretty ambiguous.